conda/recipes/dask-cuda/meta.yaml

```diff
@@ -27,8 +27,8 @@ requirements:
     - setuptools
   run:
     - python
-    - dask>=2021.09.1
-    - distributed>=2021.09.1
+    - dask>=2021.09.1,<=2021.11.2
+    - distributed>=2021.09.1,<=2021.11.2
```

Comment:
I don't think we still support `2021.09.1`, do we? For instance, `2021.11.0` will not work properly with UCX (see https://github.com/dask/distributed/pull/5505). IMO, we should pin to `2021.11.1`.

cc @quasiben @jakirkham in case you have thoughts.

Reply:
As we discussed offline, we'll pin `>=2021.11.1,<=2021.11.2`.

Reply:
```suggestion
    - dask>=2021.11.1,<=2021.11.2
    - distributed>=2021.11.1,<=2021.11.2
```